docs/extend/plugins_graphdriver.md

```diff
@@ -1,12 +1,34 @@
-# Experimental: Docker graph driver plugins
+---
+title: "Graphdriver plugins"
+description: "How to manage image and container filesystems with external plugins"
+keywords: "Examples, Usage, storage, image, docker, data, graph, plugin, api"
+---
+
+<!-- This file is maintained within the docker/docker Github
+     repository at https://github.com/docker/docker/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+
+## Changelog
+
+### 1.13.0
+
+- Moved out of experimental to stable
+
+# Docker graph driver plugins
 
 Docker graph driver plugins enable admins to use an external/out-of-process
 graph driver for use with Docker engine. This is an alternative to using the
 built-in storage drivers, such as aufs/overlay/devicemapper/btrfs.
 
 A graph driver plugin is used for image and container filesystem storage, as such
-the plugin must be started and available for connections prior to Docker Engine
-being started.
+the plugin must be enabled and available for connections prior to Docker Engine
```

Comment:
This is an awkward wording. I don't really understand what it is talking about; Is there a driver back-end that has to be running before Docker Engine can be started?